accelerators/federated-learning/fl_arc_k8s/README.md

```diff
@@ -0,0 +1,20 @@
+# Simple example of using shrike Federated Learning API + Arc + Kubernetes + AML to submit a Federated Learning pipeline experiment
+
+## Setup
+
+- Clone the current repository and set `accelerators\federated-learning\fl_arc_k8s` as your working directory.
```

Comment:
Suggestion: create a Conda environment file capturing these steps. Then the guidelines are as simple as `conda env create --file environment.yml`.

```yaml
name: fl-shrike-examples-env-py38

dependencies:

- python=3.8
- pip
- pip:
  - -r requirements.txt
```